### PR TITLE
Ascent Cloud Territory Planner Custom Connector

### PR DESCRIPTION
**Pull request by simongroom**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by simongroom** [comment]
@microsoft-github-policy-service agree [company="Ascent Cloud"]

File: certified-connectors/ascentcloudterritoryplanner/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+The Territory Planner connector from Ascent Cloud allows integration with Ascent Cloud's Territory Planner solution.
+
+The connector can be used in PowerApp and Power Automate to trigger territory plan data fetching.
+
+## Pre-requisites
+
+You will need:
+ * Active Microsoft Power Automate or Power Apps account
+ * A Territory Planner license and application subscription
+ * The organization Id and API token linked to you Territory Planner application subscription and organization
```
Comment (Amjed-Ayoub) [inline certified-connectors/ascentcloudterritoryplanner/README.md:12]: @simongroom thank you so much for your submission. All files should be free of grammatical errors and typos. in this paragraph you have "...API token linked to you Territory Planner..." please change to "...API token linked to **your** Territory Planner...".
Your readme file is missing some required fields. please see our [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) and for code raw view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md). Please include all required sections.

File: certified-connectors/ascentcloudterritoryplanner/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Territory Plan",
+        "description": "Get Territory Plan",
+        "operationId": "public_v1_get_territory_plan",
```
Comment (Amjed-Ayoub) [inline certified-connectors/ascentcloudterritoryplanner/apiDefinition.swagger.json:113]: @simongroom all operationIds should be in Camel case. please change to "PublicV1GetTerritoryPlan". Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).

File: certified-connectors/ascentcloudterritoryplanner/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+                    }
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Territory Plan",
+        "description": "Get Territory Plan",
```
Comment (Amjed-Ayoub) [inline certified-connectors/ascentcloudterritoryplanner/apiDefinition.swagger.json:112]: @simongroom all operations descriptions should be different than the summary. May I ask you please to change the description.

File: certified-connectors/ascentcloudterritoryplanner/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+                    }
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Territory Data Point Assignments",
+        "description": "Get Territory Data Point Assignments",
```
Comment (Amjed-Ayoub) [inline certified-connectors/ascentcloudterritoryplanner/apiDefinition.swagger.json:196]: @simongroom all operations descriptions should be different than the summary. May I ask you please to change the description.

File: certified-connectors/ascentcloudterritoryplanner/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+        "operationId": "public_v1_get_territory_data_point_assignments",
```
Comment (Amjed-Ayoub) [inline certified-connectors/ascentcloudterritoryplanner/apiDefinition.swagger.json:197]: @simongroom all operationIds should be in Camel case. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Comment by Amjed-Ayoub** [comment]
@simongroom Please remove the icon from your connector files as it is not required in the open source repository. Thank you so much.

File: certified-connectors/ascentcloudterritoryplanner/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,235 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Ascent Cloud Territory Planner",
+    "description": "Ascent Cloud Territory Planner Connector.",
```
Comment (Amjed-Ayoub) [inline certified-connectors/ascentcloudterritoryplanner/apiDefinition.swagger.json:5]: @simongroom please provide a proper description for your connector.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello Partners,
I hope you are doing well.
I am following up with you to see how things are going with preparing your connector.
Please let us know if you have any questions.
Thank you.

**Comment by Amjed-Ayoub** [comment]
Hello @simongroom,
I hope you are doing well.
I am following up on my messages to see if you need any help with updating your connector artifacts.
Please do not hesitate to reach out to me if you have any questions as we are always here to help. 😊
I look forward to hearing from you.
Thank you very much.

**Comment by Amjed-Ayoub** [comment]
Hello @simongroom,
This is to follow up on the status of your connector. I would like to offer help to you if you are having any issues with your connector.
Please do not hesitate to reach out if you have any questions as we are always happy to help.
Thank you very much for working with us.

**Comment by simongroom** [comment]
Thanks for following up @Amjed-Ayoub we decided not to move forward with the public connector for now.

**Comment by Amjed-Ayoub** [comment]
@simongroom thank you so much for the update. Please let us know if we can help in any way.

**Pull request closed without merging**